Police-Data-Accessibility-Project/data-sources-app
# refactor: search results and quick search

Will complete #185 when merged to `main`.

To test:
- Run the app.
- Ensure that the `QuickSearchForm` displays per updates described in #185.
- Search for data sources, ensure the search page displays per updates described in #185.
